airflow/ti_deps/deps/prev_dagrun_dep.py

```diff
@@ -19,6 +19,7 @@
 
 from sqlalchemy import func
 
+from airflow.models import DagRun
```

Comment:
```suggestion
from airflow.models.dagrun import DagRun
```

Reply:
Same applies for all imports, please import from the specific module, not `airflow.models`.

Reply:
Good catch!